README.md

```diff
@@ -1,2 +1,53 @@
-BUDGETLY APP
-main branch
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budgetly App
+
+> A web application that track your personal expenses and helps you categorize them.
+
+
+
+## Built With
+
+- HTML & CSS
+- Ruby on Rails
+- Rubocop
+- Rspec
+- OOP
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- copy this link `https://github.com/karelvanoordt/budgetly-app.git`.
+- get the directory that you want to clone the repository.
+- open the command prompt in this directory.
+- write `git clone https://github.com/karelvanoordt/budgetly-app.git`.
+- go to the repository folder in your command prompt.
+- run `bundle install`.
+- run `npm install`
+- create database in your postgre called `recipe_development`
+- run `rails db:migrate`
+- To run the test run `rspec` in your command prompt.
+- To run the rails server run  `rails s` in your command prompt.
+
+## Live Demo
+
+[Click here to check the live demo](https://desolate-peak-95618.herokuapp.com/)
+
+## Loom Video Presentation
+[Click here to watch the demo presentation](https://www.loom.com/share/23dc43c44bb445778e636a0c295f34e9)
+
+## Author
+
+👤 **Karel van Oordt**
+
+- GitHub: [@karelvanoordt](https://github.com/karelvanoordt)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/karelvanoordt/)
+
+
+**Aknowledgment**
+
+- Big thanks to [Gregoire Vella](https://www.behance.net/gregoirevella) for providing the design guidelines for this project.
+
+
```

Comment:
### OPTIONAL
Nice to add a design link in the acknowledgment section. 
You can fix the typo `Aknowledgment`
- [ ] Please also create a license section and add [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)

[Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=The%20Creative%20Commons%20license%20of%20the%20design%20requires%20that%20you%20give%20appropriate%20credit%20to%20the%20author.%20Therefore%2C%20you%20must%20do%20it%20in%20the%20README%20of%20your%20project.)